metrics/web/test/base/presentation/widgets/svg_image_test.dart

```diff
@@ -0,0 +1,351 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/widgets/svg_image.dart';
+import 'package:metrics/util/renderer_helper.dart';
+import 'package:mockito/mockito.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("SvgImage", () {
```

Comment:
Should we test that this widget applies the default `rendererHelper` if null is passed?